history/characters/340000_night_elf.txt

```diff
@@ -2155,13 +2155,14 @@ arantir={
 	715.1.1={ death=yes }
 }
 
-#dynasty = night_elf1
-night_elf1 = { #Generated
-	name = Wifinnyia
+#dynasty = dynn_swiftarrow
+night_elf1 = { # Warsong Gulch Battlemaster
+	name = Su'ura
 	female = yes
-	dynasty = night_elf1
+	dynasty = dynn_swiftarrow # note - reuse autumnshade somewhere
 	religion = kaldorei_religion
 	culture = night_elf
+	trait=education_martial_4 trait=
```

Comment:
"trait="

Reply:
fuuuuck ok will finish her

Reply:
ok fixed apparently this character was already in game